examples/lightrag_agent_(developing)/services/rag_service.py

```
@@ -0,0 +1,468 @@
+                mode=mode,
+                top_k=top_k or self.config.top_k,
+                chunk_top_k=self.config.chunk_top_k,
+                max_entity_tokens=0,
+                max_relation_tokens=0,
+                max_total_tokens=self.config.max_total_tokens,
+                enable_rerank=self.config.enable_rerank,
+                **kwargs
+            )
+            
+            response = await self.lightrag.aquery(query, param=params)
+            return response
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)